Makefile

```diff
@@ -492,6 +493,8 @@ test-e2e-setup: login-required
 	VSL_REGION="$(VSL_REGION)" \
 	BSL_REGION="$(BSL_REGION)" \
 	BSL_AWS_PROFILE="$(BSL_AWS_PROFILE)" \
+	OC_CLI="$(OC_CLI)" \
```

Comment:
These env vars are not needed anymore here, right?

Reply:
Right, the *_settings.sh scripts don't need either of these.